modules/navier_stokes/src/kernels/INSBase.C

```diff
@@ -276,8 +276,12 @@ INSBase::tau()
   Real nu = _mu[_qp] / _rho[_qp];
   RealVectorValue U(_u_vel[_qp], _v_vel[_qp], _w_vel[_qp]);
   Real h = _current_elem->hmax();
-  return _alpha / std::sqrt((2. * U.norm() / h) * (2. * U.norm() / h) +
-                            9. * (4. * nu / (h * h)) * (4. * nu / (h * h)));
+  if (!_transient_term)
+    return _alpha / std::sqrt((2. * U.norm() / h) * (2. * U.norm() / h) +
+                              9. * (4. * nu / (h * h)) * (4. * nu / (h * h)));
+  else
+    return _alpha / std::sqrt((1. / _dt) * (1. / _dt) + (2. * U.norm() / h) * (2. * U.norm() / h) +
+                              9. * (4. * nu / (h * h)) * (4. * nu / (h * h)));
```

Comment:
This (and especially in the Jacobian) is a lot of duplicated code just to turn off the one contribution...
```
Real transient_part = _transient_term ? 4. / (_dt * _dt) : 0.;
return _alpha / std::sqrt(transient_part + ...);
```
seems cleaner.

Reply:
The above also switches to the more common 2/dt form.